profiler/LICENSE

```diff
@@ -0,0 +1,201 @@
+﻿                                 Apache License
```

Comment:
We already have a LICENSE at the root so you should be fine to remove this

Reply:
removed